pkg/ibmcloud/api.go

```diff
@@ -373,13 +373,27 @@ func createPolicy(token, accountID, iamID, serviceName, serviceInstance, role st
 
 	err = postBody(policyEndpoint, header, nil, body, &result)
 	if err != nil {
-		fmt.Println(err) //TODO delete after debugging
 		return nil, err
 	}
 
 	return &result, nil
 }
 
+func isMemberOfAccessGroup(token, accessGroupID, iamID string) error {
+
+	header := map[string]string{
+		"Authorization":         "Bearer " + token,
+	}
+
+	checkMembershipEndpoint := iamEndpoint + "/" + accessGroupID + "/members/" + iamID
+	err := head(checkMembershipEndpoint, header, nil, nil)
```

Comment:
thats an interesting api. so its not a get but head?

Reply:
Yeah, returns 204 if user is a member of access group or 404 if they are not a member. 
https://cloud.ibm.com/apidocs/iam-access-groups#is-member-of-access-group

I think we need to take a look at how errors are handled in the application so that a 404 here doesn't get returned as a generic 500 internal server error in our api.